source/blog/2018-11-30-the-emberjs-times-issue-75.md

```diff
@@ -1,19 +1,23 @@
 ---
 title: The Ember Times - Issue No. 75
-author: the crowd
+author: Kenneth Larsen
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/2018/11/30-the-ember-times-issue-75.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+📖Get a sneak peek on the new ember-cli guides
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [New Guides For Ember-CLI](https://cli.emberjs.com/release/)
+There's a **new and improved version of the ember-cli guides out**. [@jenweber](https://github.com/jenweber) has been championing moving the ember-cli guides into the ember domain with enhanced content. 
+
+There will be a more formal launch of the new ember-cli guides on the Ember-blog but since you're **an Ember Times subscriber**, [you get a sneak peek](https://cli.emberjs.com/release/).
```

Comment:
```suggestion
There will be a more formal launch of the new Ember CLI guides on the [Ember blog](https://emberjs.com/blog/) but since you're **an Ember Times subscriber**, [you get a sneak peek](https://cli.emberjs.com/release/).
```